common/src/main/java/org/apache/rocketmq/common/UtilAll.java

```diff
@@ -604,7 +606,7 @@ public static byte[] getIP() {
                             if (ipCheck(ipByte)) {
                                 if (!isInternalIP(ipByte)) {
                                     return ipByte;
-                                } else if (internalIP == null) {
+                                } else if (internalIP == null || internalIP[0] == (byte) 127) {
```

Comment:
This judgment changes the behavior of the getIP function. If we only have lo interface, getIP will throw RuntimeException.

Reply:
If we only have lo interface,`internalIP == null || internalIP[0] == (byte) 127)` will be true，so it will set `internalIP` to localhost and return.
If we have lo and LAN ip，when it find the LAN ip，`internalIP == null || internalIP[0] == (byte) 127)` also will be true, so it will override the value of `internalIP`.
Am I missing something？

Reply:
You are right. Please ignore this review.